### PR TITLE
Troyfeng116/pipeline demo

### PR DESCRIPTION
**Pull request by troyfeng116**

### CODE REVIEW

**Comment by troyfeng116** [comment]
@niansong1996 Once this is merged, we can update the colab notebook with something like the pipeline code in the README. Let me know what you think

File: pipeline/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from model.single_doc import LexRankModel
 
 from dataset.st_dataset import SummDataset
-from dataset.non_huggingface_datasets import ScisummnetDataset
+from dataset import ScisummnetDataset
```
Comment (niansong1996) [inline pipeline/__init__.py:6]: Why does this depend on a specific dataset here?

Comment (troyfeng116) [reply pipeline/__init__.py:6]: `ScisummnetDataset` data instances are formatted a little strangely, it's a single-doc dataset but each instance source is a list containing two items: the document xml and citations in json format or something like that. See line 357 in `dataset/dataset_loaders.py` (the `_process_data` function for `Scisummnet`). Not sure if this was intentional, I just did a bandaid patch on it for now assuming we might fix it later

Comment (niansong1996) [reply pipeline/__init__.py:6]: Okay, let's fix that in another thread. Merging this one now

**Review by niansong1996 — COMMENTED** [review]
LGTM, merging it now

**Pull request merged**